src/utils/env/getClientSideEnvs.ts

```diff
@@ -0,0 +1,13 @@
+export function getClientSideEnvs() {
+  const whitelist: string[] = ['PENUMBRA_CHAIN_ID', 'PENUMBRA_CUILOA_URL'];
```

Comment:
suggestion: add `.env.example` file to env. It shouldn't have any values, just keys with comments on what they do, and where to get them from